<ui version="4.0" >
 <class>DiagnosticsDialog</class>
 <widget class="QDialog" name="DiagnosticsDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>379</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Diagnostics</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QTabWidget" name="tabWidget" >
       <property name="minimumSize" >
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="autoFillBackground" >
        <bool>false</bool>
       </property>
       <property name="currentIndex" >
        <number>0</number>
       </property>
       <widget class="QWidget" name="scrobblingTab" >
        <attribute name="title" >
         <string>Scrobbling</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QGridLayout" >
           <item row="0" column="1" >
            <widget class="QLabel" name="submissionServerStatusLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Checking...</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="submissionServerLabel" >
             <property name="text" >
              <string>Submission server status:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLabel" name="lastConnectionStatusLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Unknown</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="lastConnectionLabel" >
             <property name="text" >
              <string>Last successful connection:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="cachedTracksTitle" >
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string>Cached Tracks</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="cachedTracksList" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="rootIsDecorated" >
              <bool>false</bool>
             </property>
             <property name="uniformRowHeights" >
              <bool>true</bool>
             </property>
             <property name="itemsExpandable" >
              <bool>false</bool>
             </property>
             <property name="allColumnsShowFocus" >
              <bool>true</bool>
             </property>
             <property name="columnCount" >
              <number>3</number>
             </property>
             <column>
              <property name="text" >
               <string>Artist</string>
              </property>
             </column>
             <column>
              <property name="text" >
               <string>Track</string>
              </property>
             </column>
             <column>
              <property name="text" >
               <string>Album</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cachedTracksLabel" >
             <property name="enabled" >
              <bool>false</bool>
             </property>
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Checking cache...</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="reconnect" >
           <property name="text" >
            <string>Reconnect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="radioTab" >
        <attribute name="title" >
         <string>Radio</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QLabel" name="radioServerLabel" >
             <property name="text" >
              <string>Radio server status:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="radioServerStatusLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Checking...</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QGridLayout" >
           <item row="2" column="2" >
            <widget class="QLabel" name="outputBufferLabel" >
             <property name="minimumSize" >
              <size>
               <width>35</width>
               <height>0</height>
              </size>
             </property>
             <property name="text" >
              <string>0</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0" >
            <widget class="QLabel" name="label_10" >
             <property name="text" >
              <string>Output buffer:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2" >
            <widget class="QLabel" name="httpBufferLabel" >
             <property name="minimumSize" >
              <size>
               <width>35</width>
               <height>0</height>
              </size>
             </property>
             <property name="text" >
              <string>0</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" >
            <widget class="QLabel" name="label_8" >
             <property name="text" >
              <string>HTTP buffer:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QProgressBar" name="httpBufferProgress" >
             <property name="value" >
              <number>0</number>
             </property>
             <property name="textVisible" >
              <bool>false</bool>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QProgressBar" name="decodedBufferProgress" >
             <property name="value" >
              <number>0</number>
             </property>
             <property name="textVisible" >
              <bool>false</bool>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="2" >
            <widget class="QLabel" name="decodedBufferLabel" >
             <property name="minimumSize" >
              <size>
               <width>35</width>
               <height>0</height>
              </size>
             </property>
             <property name="text" >
              <string>0</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="1" >
            <widget class="QProgressBar" name="outputBufferProgress" >
             <property name="value" >
              <number>0</number>
             </property>
             <property name="textVisible" >
              <bool>false</bool>
             </property>
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="label_9" >
             <property name="text" >
              <string>Decoded buffer:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="fingerprintingTab" >
        <attribute name="title" >
         <string>Fingerprinting</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <property name="spacing" >
          <number>-1</number>
         </property>
         <item>
          <layout class="QGridLayout" >
           <item row="0" column="0" >
            <widget class="QLabel" name="submissionServerLabel_2" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string>Tracks in queue:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" >
            <widget class="QLabel" name="fpQueueSizeLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" >
            <widget class="QLabel" name="submissionServerLabel_3" >
             <property name="text" >
              <string>Currently fingerprinting:</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="indent" >
              <number>-1</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1" >
            <widget class="QLabel" name="fpCurrentTrackLabel" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text" >
              <string/>
             </property>
             <property name="alignment" >
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
             </property>
             <property name="wordWrap" >
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" >
           <property name="spacing" >
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="fingerprintedTracksTitle" >
             <property name="font" >
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text" >
              <string>Recently fingerprinted tracks</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QTreeWidget" name="fingerprintedTracksList" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="rootIsDecorated" >
              <bool>false</bool>
             </property>
             <property name="uniformRowHeights" >
              <bool>true</bool>
             </property>
             <property name="itemsExpandable" >
              <bool>false</bool>
             </property>
             <property name="allColumnsShowFocus" >
              <bool>true</bool>
             </property>
             <property name="columnCount" >
              <number>3</number>
             </property>
             <column>
              <property name="text" >
               <string>Artist</string>
              </property>
             </column>
             <column>
              <property name="text" >
               <string>Track</string>
              </property>
             </column>
             <column>
              <property name="text" >
               <string>Album</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="iPodTab" >
        <attribute name="title" >
         <string>iPod Scrobbling</string>
        </attribute>
        <layout class="QVBoxLayout" >
         <item>
          <layout class="QHBoxLayout" >
           <item>
            <widget class="QComboBox" name="iPodScrobbleType" >
             <property name="sizePolicy" >
              <sizepolicy vsizetype="Fixed" hsizetype="Expanding" >
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text" >
               <string>iTunes automatically manages my iPod</string>
              </property>
             </item>
             <item>
              <property name="text" >
               <string>I manually manage my iPod</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="scrobbleIpodButton" >
             <property name="text" >
              <string>Scrobble iPod</string>
             </property>
             <property name="autoRepeatDelay" >
              <number>300</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QListWidget" name="ipodInfoList" />
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3" >
       <property name="title" >
        <string>Logs</string>
       </property>
       <property name="flat" >
        <bool>true</bool>
       </property>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QPushButton" name="viewLogButton" >
          <property name="text" >
           <string>View Log</string>
          </property>
          <property name="autoDefault" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="refreshButton" >
       <property name="text" >
        <string>Refresh</string>
       </property>
       <property name="shortcut" >
        <string>F5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyToClipboardButton" >
       <property name="text" >
        <string>Copy to Clipboard</string>
       </property>
       <property name="shortcut" >
        <string>Ctrl+C</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton" >
       <property name="text" >
        <string>&amp;Close</string>
       </property>
       <property name="default" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
